### ISSUE TITLE
Document how to share/publish aliases

### ISSUE THREAD

**Issue by texastoland** [Homebrew/homebrew-aliases#59]
I have aliases that I find quite useful like [`tree`](https://github.com/texastoland/dotfiles/blob/main/.config/brew-aliases/tree) or a version of `bundle dump` that includes the output of `tree` in a frontmatter. Is it already possible to share them via a tap or something?

**Comment by MikeMcQuaid** [comment]
Something like this feels like a good candidate for Homebrew/homebrew-bundle to `brew bundle dump` and `brew bundle install`. Thoughts?

**Comment by texastoland** [comment]
I'm already versioning my `.config` folder where they're saved with the rest of my dotfiles. That already works great! Bundle is phenomenal for syncing Codespaces thanks for that too 🏆

My specific use case is I want to make a dotfiles course and find these aliases almost essential. It'd be convenient if I could recommend a maintainable way (besides domload/`curl`) to install them.

**Comment by texastoland** [comment]
> Something like this feels like a good candidate for Homebrew/homebrew-bundle

Or if you're suggesting upstreaming these I'd be happy to open an issue there!

**Comment by MikeMcQuaid** [comment]
> > Something like this feels like a good candidate for Homebrew/homebrew-bundle
> 
> Or if you're suggesting upstreaming the features I'd be happy to open an issue there!

Yes, please! Issue would be nice but pull request ideal. Feel free to open early/broken and we can help you get it over the line.

**Comment by texastoland** [comment]
I contributed #25 here but it was broken because I got no Ruby fu 🙈 I'll try though!

**Comment by texastoland** [comment]
Closing in favor of future proposals/PRs to brew and bundle respectively:

1. Default behavior for `brew tree` to print `leaves --installed-on-request` that are `--installed --formula` with an optional list of orphans and how to remove them
2. An optional comment header in the Brewfile with the exact output of the `brew tree --list-orphans`
